# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1260583131 18000
#      Fri Dec 11 20:58:51 2009 -0500
# Node ID cc7ea60837746fe3b7abf590101361fb80b8bf5a
# Parent  a24a528d98cc40eda6aa91f0cf8aa0a56b978688
configure.ac: use pthread flags and libs for tests

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,13 @@
+2009-12-11  John W. Eaton  <jwe@octave.org>
+
+	* configure.ac: Add $PTHREAD_LIBS to $LIBS and $PTHREAD_CFLAGS
+	to $CFLAGS for subsequent tests.
+
 2009-12-11  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: remove install-extras target and use automake syntax to
 	distribute NEWS file.
 	* examples/Makefile.am: remove install-images target and use automake
 	syntax to distribute image data.
 
 2009-12-11  John W. Eaton  <jwe@octave.org>
diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -272,16 +272,20 @@ else
     ;;
   esac
 fi
 AC_SUBST(INCLUDE_DEPS)
 AC_SUBST(DEPEND_FLAGS)
 AC_SUBST(DEPEND_EXTRA_SED_PATTERN)
 
 ACX_PTHREAD
+### Include pthread libs and flags here in case other tests need them.
+### They seem to be required for the OpenGL tests on Debian systems.
+LIBS="$PTHREAD_LIBS $LIBS"
+CFLAGS="$CFLAGS $PTHREAD_CFLAGS"
 
 AC_PATH_X
 if test "$have_x" = "yes"; then
   AC_DEFINE(HAVE_X_WINDOWS, 1, [Define if you have X11])
 
   if test "$x_includes" != "NONE"; then
     X11_INCFLAGS="$x_includes"
   fi
